src/realm/query_expression.hpp

```diff
@@ -2517,6 +2552,162 @@ class Columns<SubTable> : public Subexpr2<SubTable> {
     }
 };
 
+template <typename T>
+class ListColumns : public Subexpr2<T> {
```

Comment:
It looks like this doesn't provide any of the functionality that `Columns<StringData>` provides, meaning that `begins_with` / `ends_with` / `contains` / `like` are not available.